src/Cpanel/PluginActions.php

```diff
@@ -16,33 +16,39 @@ class PluginActions extends AbstractPluginActions
     protected $userConfig;
     protected $composer;
 
-    public static $CONFIG = array(
```

Comment:
I just realized this wasn't the backend.  cPanel is pinned to 5.6, we can use short array syntax.